test/ringpop-assert.js

```diff
@@ -910,8 +924,11 @@ module.exports = {
     waitForJoins: waitForJoins,
     waitForPingReqs: waitForPingReqs,
     waitForPing: waitForPing,
-    waitForEmptyPing: waitForEmptyPing,
-    // drainSUTDissemination: drainSUTDissemination,
+
+    // waitForEmptyPing is slow. If possible, use drainDisseminator instead.
+    // waitForEmptyPing: waitForEmptyPing,
```

Comment:
Lets remove this dead code before merging. The comment above is an actual good comment for the line below so that should stay.